src/components/data-pane/field-list.scss

```diff
@@ -6,3 +6,8 @@
 .field-list-item {
   margin-bottom: 4px;
 }
+
+.field-list-no-wildcards {
+  max-height: calc(100vh - 167px);
```

Comment:
Why `-167px`?  

Note that 167 is a prime number so I suspect that the allocated space for the debug pane would not be proportional to other size of the UI.   If there is a good reason, this is fine.  But I wanna understand how you get this number, which seems a bit arbitrary.

Reply:
159px is the total of the height, padding and margins of the header and other items in the data-pane. Then I added 8px for padding as that seems to be the convention. I can change the padding if you would like to a different number. The reason the number is odd and prime is because the header is 33px tall.

Reply:
This is fine, but better add a comment describing where 167px comes from.